src/test/kotlin/no/nav/dagpenger/joark/mottak/DummyJoarkProducer.kt

```diff
@@ -47,23 +47,22 @@ class DummyJoarkProducer(journalpostProducerProperties: Properties) {
 
                 """.trimIndent()
 
-    fun produceDummyMessage() {
-        val id = Random().nextInt(2000).toString()
+    fun produceEvent(journalpostId: Long, tema: String) {
         val avroSchema = Schema.Parser().parse(schemaSource)
         val joarkJournalpost = GenericData.Record(avroSchema)
-        joarkJournalpost.put("hendelsesId", id)
-        joarkJournalpost.put("versjon", id.toInt())
-        joarkJournalpost.put("journalpostId", id.toLong())
+        joarkJournalpost.put("hendelsesId", journalpostId.toString())
```

Comment:
This code can be Kotlinified by doing:

```suggestion
        joarkJournalpost.apply {
          "hendelsesId" to journalpostId.toString()
        }
```

Reply:
ah, sure - nice, will apply :)